src/ate/proto/dut_commands.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Contains the definitions of JSON messages used to communicate with the
+// provisioning firmware running on CP and FT test stages.
+
+syntax = "proto3";
+
+package ot.dut_commands;
+
+option go_package = "dut_commands_go_pb";
+
+// Provides the manuf_cp_provisioning_data data structure used by
+// provisioning firmware.
+message CpProvisioningDataJSON {
```

Comment:
nit: In a follow up, it may be nice to drop links to where these are defined in the opentitan code base (i.e., in provisioning_data.h)